catalog/docs/src/main/resources/_contents/integrating-contents.adoc

```diff
@@ -1394,9 +1394,296 @@ Schematron files may reference other schematron files using an include statement
 However, when using the document function within a schematron ruleset to reference another file, the path must be absolute or relative to the ${ddf-branding} installation home directory.
 ====
 
-===== Configuring
+==== Configuring
 
 Schematron validation services are configured with a namespace and one or more schematron rule sets.
 Additionally, warnings may be suppressed so that only errors are reported.
 To create a new service, ensure that catalog-schematron-plugin is started and then click Schematron Validation Services.
 
+=== Adding New Attribute Types, Metacard Types, and Validators Using JSON Files
+
+[WARNING]
+====
+This section concerns capabilities that are considered experimental. The features described in this section may change or be removed in a future version of the application.
+====
+
+=== Definition File Format
```

Comment:
this should be a subsection to "Adding New Attribute Types, Metacard Types, and
Validators Using JSON Files"
eg: 
`+==== Definition File Format`